META.json

```diff
@@ -2,7 +2,7 @@
    "name": "pg_repack",
    "abstract": "PostgreSQL module for data reorganization",
    "description": "Reorganize tables in PostgreSQL databases with minimal locks",
-   "version": "1.4.8",
+   "version": "1.4.9",
```

Comment:
I would like to separate the update of support for PG 16 and the cut of a new release. These two do not necessarily have to go into a single PR.